sound/soc/sof/pm.c

```diff
@@ -318,10 +318,13 @@ static int sof_suspend(struct device *dev, bool runtime_suspend)
 	/* notify DSP of upcoming power down */
 	ret = sof_send_pm_ipc(sdev, SOF_IPC_PM_CTX_SAVE);
 	if (ret < 0) {
+		/*
+		 * DSP will be powered off right after.
+		 * So just report the error and continue.
```

Comment:
That sounds like kicking the can down the read. If the IPC is not successful, doesn't it indicate trouble ahead on resume?

Reply:
@plbossart this PR is by no means meant to address the underlying problem. I am only avoiding the problem that occurs when this IPC timesout for reasons not known so far. So it is only a workaround until the root cause is found. We're seeing repeated IPC timeouts on this particular iPC and if it times out, we reach a point of no recovery today.